staging-testnet.yaml

```diff
@@ -1,36 +1,30 @@
 args:
   configurationKey:
     tag: NixStr
-    contents: mainnet_dryrun_full
+    contents: bench
```

Comment:
this should go into a new file, and the original content restored

Reply:
Done.